### PR TITLE
chore(fmg): fmg documentation

### PR DESCRIPTION
**Pull request by atarashansky**
- #3623 

### Reviewers
**Functional:** 

**Readability:** 

---

## Changes
- add
- remove
- modify

### CODE REVIEW

File: frontend/src/pages/docs/[...slug].tsx [hunk 1]
```diff
@@ -8,6 +8,8 @@ import {
   MDXRemoteProps,
   MDXRemoteSerializeResult,
 } from "next-mdx-remote";
+import rehypeMathJax from 'rehype-mathjax';
+import remarkMath from "remark-math";
```
Comment (seve) [inline frontend/src/pages/docs/[...slug].tsx:12]: Shouldn't there be a package.json diff?

**Review by seve — CHANGES_REQUESTED** [review]

File: frontend/src/pages/docs/[...slug].tsx (hunk 1 above)
Comment (atarashansky) [reply frontend/src/pages/docs/[...slug].tsx:12]: Definitely added it to the commit, not sure why it's not there...trying again

**Comment by signechambers1** [comment]
@atarashansky would you mind adding the additional caveat that we do not calculate markers for cell types with <25 cells or blood?

Otherwise looks good to me. Tagging @ambrosejcarr for review as well.

**Comment by atarashansky** [comment]
> @atarashansky would you mind adding the additional caveat that we do not calculate markers for cell types with <25 cells or blood?
> 
> Otherwise looks good to me. Tagging @ambrosejcarr for review as well.

It's already in there, the last bullet point!

**Review by ambrosejcarr — APPROVED** [review]
LGTM. Thanks for flagging me for review.

**Pull request merged**